src/components/Login.jsx

```diff
@@ -178,7 +178,7 @@ class Login extends Component {
         <Row>
           <Col md={7} className="login-description">
             <div className="login-heading">
-              App Name
+              Social Distribution
```

Comment:
Can we change the title in index.html as well so it doesn't say React App on the browser tab.

Reply:
Just did😄